examples/fabric-bridge-app/linux/RpcServer.cpp

```diff
@@ -62,6 +63,9 @@ pw::Status FabricBridge::AddSynchronizedDevice(const chip_rpc_SynchronizedDevice
         return pw::Status::Unknown();
     }
 
+    // Ignoring the error returned.
```

Comment:
uh...what error gets returned? Why are you ignoring it? Is it real?

Reply:
Changed to VerifyOrDie, if we crash we will fix any assumptions.